.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "third_party/uvwasi"]
 	path = third_party/uvwasi
 	url = https://github.com/nodejs/uvwasi
+[submodule "third_party/picosha2"]
+	path = third_party/picosha2
+	url = https://github.com/okdshin/PicoSHA2
```

Comment:
Doesn't adding this dependency make much harder to compile wasm2c output?   Seems like a fairly be downside.

Reply:
Hmm, I don't think so -- it's only used by the wasm2c tool, not its output. (To be clear, in this PR at present, libwabt can be compiled either to link against libcrypto or with this vendored implementation, and if doing the latter, it just gets compiled in; it doesn't create a dependency.)

There isn't an added dependency for the wasm2c output, the runtime, or the host.

Reply:
Thanks for clarifying, that sounds much more reasonable.